.changeset/olive-buttons-kiss.md

```diff
@@ -0,0 +1,10 @@
+---
+'@crowdstrike/ember-toucan-core': patch
+'@crowdstrike/ember-toucan-form': patch
+---
+
+Both `Textarea` and `Input` Controls are now full width by default.
+
+The `Textarea` Control markup was adjusted to account for collision with the resize handle and focus/error shadows.
+
+The `Input` Control now has small padding along the x-axis.
```

Comment:
These changes are arguably `minor` aka "breaking" since we are pre-1.0; however, it appears most folks are using the Field components rather than Controls directly. I think the correct thing to do here may be to choose `minor`, but if there is hesitancy in creating a breaking change release so soon, a `patch` will do.  Any thoughts?

Reply:
It's a visual change which seems to indicate [it's minor](https://medium.com/eightshapes-llc/visual-breaking-change-in-design-systems-1e9109fac9c4), but it's pretty subtle. I would not be able to tell if you didn't do the side by side...so I would say `patch` in this case?

Reply:
Works for me 👍